client/src/partials/fiscal/fiscal.js

```diff
@@ -0,0 +1,105 @@
+angular.module('bhima.controllers')
+.controller('FiscalController', FiscalController);
+
+FiscalController.$inject = ['$state', 'FiscalService', '$interval', 'ModalService', 'NotifyService'];
+
+/**
+
+ */
+function FiscalController($state, fiscalService, $interval, ModalService, Notify) {
+  var vm = this;
+  var today = new Date(); 
+
+  // pagination configuration
+  /** @todo this should all be moved to a component */
+  vm.pageSize     = 5;
+  vm.currentPage  = 1;
+  vm.fiscalYears  = [];
+  vm.maxSize      = 5;
+  vm.del          = del;
+
+  vm.sort = sort;
+  vm.state = $state;
+
+  vm.sortByName = "ASC";
```

Comment:
All these variables look like they can be put in a service or component.  We should take a look at this for a second iteration on this module.